rasa/core/actions/forms.py

```diff
@@ -550,13 +551,21 @@ async def _ask_for_slot(
     ) -> List[Event]:
         logger.debug(f"Request next slot '{slot_name}'")
 
-        action_to_ask_for_next_slot = action.action_from_name(
-            self._name_of_utterance(domain, slot_name), domain, self.action_endpoint
+        action_to_ask_for_next_slot = self._name_of_utterance(domain, slot_name)
+        if not action_to_ask_for_next_slot:
+            # Use a debug log as the user might have asked as part of a custom action
+            logger.debug(
+                f"There was no action found to ask for slot '{slot_name}' "
```

Comment:
this sounds like something unexpected, do we expect a developer to fix this?

Reply:
@wochinge could you please take this over

Reply:
It chose `debug` because it could be that a user already dispatched a message to ask for the slot via the custom action for validation.